src/CommonTreeDropdown.php

```diff
@@ -403,7 +403,7 @@ public function post_updateItem($history = 1)
             $newParentNameID = '';
 
             $parent = clone $this;
-            if ($oldParentID > 0) {
+            if (!$this->isNewID($oldParentID)) {
```

Comment:
This could be used to validate that object really exists. Checking `!isNewID()` works here, but does not seems a relevant test from human POV and could be confusing.
```suggestion
            if ($parent->getFromDB($oldParentID)) {
```